ixmp/core.py

```diff
@@ -312,6 +314,25 @@ def add_region_synomym(self, region, mapped_to):
         else:
             _logger_region_exists(_regions, region)
 
+    def check_access(self, user, models, access="view"):
+        """Check access to specific model
+
+        :param user: registered user name
```

Comment:
please follow the docstrings convention used for other functions

```
Parameters
----------
user: str
    user name
```

Reply:
done